syncer/syncer.go

```diff
@@ -367,6 +367,8 @@ func (s *Syncer) syncLayer(ctx context.Context, layerID types.LayerID) (*types.L
 		if err = s.mesh.SetZeroBlockLayer(layerID); err != nil {
 			s.logger.WithContext(ctx).With().Panic("failed to set zero-block for genesis layer", layerID, log.Err(err))
 		}
+	}
+	if !layerID.After(types.GetEffectiveGenesis()) {
```

Comment:
should be the opposite?

Reply:
i think it is correct, why opposite? e.g fetch data from db `if layerID <= genesis`

Reply:
i misread. sorry